# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/PWebrtcTCPSocket.ipdl
# Commit: b83124cf3a89
# Full Hash: b83124cf3a891091773bcfe2cd15e36155fc5a2d
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Teach WebrtcTCPSocket to use TLS when configured to. r=mjf,mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45103
# ==============================================================================

diff -r b879a0044cf3 -r b83124cf3a89 media/mtransport/ipc/PWebrtcTCPSocket.ipdl
--- a/media/mtransport/ipc/PWebrtcTCPSocket.ipdl	Tue Sep 17 18:17:18 2019 +0000
+++ b/media/mtransport/ipc/PWebrtcTCPSocket.ipdl	Tue Sep 17 18:17:24 2019 +0000
@@ -23,6 +23,7 @@
                   int32_t aPort,
                   nsCString aLocalAddress,
                   int32_t aLocalPort,
+                  bool aUseTls,
                   LoadInfoArgs aLoadInfoArgs,
                   nsCString aAlpn,
                   int proxyPolicy);